channeldb/channel.go

```diff
@@ -849,6 +868,84 @@ func fetchChanNumUpdates(openChanBucket *bolt.Bucket, channel *OpenChannel) erro
 	return nil
 }
 
+func putChanSatoshisSent(openChanBucket *bolt.Bucket, channel *OpenChannel) error {
```

Comment:
The `[put | delete | fetch]ChanSatoshisSent` functions along with the satoshis received variants aren't actually needed. There's already the `[put | delete | fetch]ChanTotalFlow` functions that handle reading/writing the channel meta-data to disk.